test/kata5.reachDestination.test.js

```diff
@@ -1,5 +1,9 @@
 const { reachDestination } = require("../src");
 
 describe("reachDestination", () => {
-  test("returns string with estimated time of arrival", () => {});
-});
+  test("returns string with estimated time of arrival", () => {
+    expect(reachDestination(44, 10)).toEqual(
+      "I will be there in 4.5, see you soon."
+    );
+  });
```

Comment:
As previously mentioned, you should add the units for time to this string. After doing this, I would add a separate test for your edge case where the arrival time is `1` e.g.
```
test("returns a string with the estimated time of arrival for 1 hour", () => {
    expect(reachDestination(1, 1)).toEqual("I will be there in 1 hour, see you soon.");
  });
```